pallets/interest-accrual/src/lib.rs

```diff
@@ -480,28 +468,12 @@ impl<T: Config> InterestAccrual<T::InterestRate, T::Balance, Adjustment<T::Balan
 		Pallet::<T>::do_renormalize_debt(old_interest_rate, new_interest_rate, normalized_debt)
 	}
 
-	fn reference_yearly_rate(
-		interest_rate_per_year: T::InterestRate,
-	) -> Result<T::InterestRate, DispatchError> {
-		Pallet::<T>::reference_yearly_interest_rate(interest_rate_per_year)
+	fn reference_rate(interest_rate_per_year: T::InterestRate) -> sp_runtime::DispatchResult {
+		Pallet::<T>::reference_interest_rate(interest_rate_per_year)
 	}
 
-	fn reference_rate(interest_rate_per_sec: T::InterestRate) -> Result<(), DispatchError> {
-		Pallet::<T>::reference_interest_rate(interest_rate_per_sec)
-	}
-
-	fn unreference_rate(interest_rate_per_sec: T::InterestRate) -> Result<(), DispatchError> {
-		Pallet::<T>::unreference_interest_rate(interest_rate_per_sec)
-	}
-
-	fn convert_additive_rate_to_per_sec(
-		interest_rate_per_year: T::InterestRate,
-	) -> Result<T::InterestRate, DispatchError> {
-		Pallet::<T>::validate_rate(interest_rate_per_year)?;
-		let interest_rate_per_sec = interest_rate_per_year
-			.checked_div(&T::InterestRate::saturating_from_integer(SECONDS_PER_YEAR))
-			.ok_or(ArithmeticError::Underflow)?;
-		Ok(interest_rate_per_sec)
```

Comment:
This method is no longer needed because the addition is now at year level. The resulting rate will be referenced as before and there evaluated by `validate_rate`.